addons/crm/models/crm_lead.py

```diff
@@ -1842,7 +1858,8 @@ def _prepare_customer_values(self, partner_name, is_company=False, parent_id=Fal
             'is_company': is_company,
             'type': 'contact'
         }
-        if self.lang_id:
+        # Add language if present and active
+        if self.lang_id and self.env['res.lang']._lang_get(self.lang_id.code):
```

Comment:
can't we just check `self.lang_id.active` ?

Reply:
Yes, thanks. Field being an empty record set when the relation is not set is great!